### DISCUSSION TITLE
hi!

### DISCUSSION THREAD

**Discussion by vodkawrath** [community/community#18910] [Other Feature Feedback, Questions, & Ideas] [answered]
i'm new here, is pt-br allowed? i'm finally started few days ago learning and i loving it

**Comment by S2Sofficial** [comment]
Hi, yes i think so!
 
Glad you're loving it.

**Reply by vodkawrath** [reply]
ah, então posso me expressar melhor, obrigado

**Reply by S2Sofficial** [reply]
Great, you can also check my profile to get started or there are lots of templates available.

**Reply by vodkawrath** [reply]
ok! vou dar uma olhada, obrigado desde ja!

**Comment by Nik-08** [comment, marked as answer]
For more info, see: https://github.com/Schweinepriester/github-profile-achievements/
